src/main/java/nextstep/subway/domain/FareCalculatorImpl.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.subway.domain;
+
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FareCalculatorImpl implements FareCalculator {
+    private static final String DISTANCE_OVER = "distanceOver";
+    private static final String DISTANCE_UNDER = "distanceUnder";
+    private static final String FARE = "fare";
+    private static final String STEP = "step";
+
+    private static final int DEFAULT_FARE = 1250;
```

Comment:
기본 요금 상수 👍 
지금은 숫자가 작아서 가독성에 무리가 없지만 숫자가 커지는 경우 `_`를 활용해 가독성을 높여줄 수 있습니다
```suggestion
    private static final int DEFAULT_FARE = 1_250;
```